src/plugins/janus_audiobridge.c

```diff
@@ -8777,9 +8779,61 @@ static void *janus_audiobridge_participant_thread(void *data) {
 					jitter_ticks = 0;
 				}
 				jitter_buffer_tick(participant->jitter);
+
 				if(ret != JITTER_BUFFER_OK) {
-					/* No packet in the jitter buffer? Move on the talking detection, if needed */
-					janus_audiobridge_participant_istalking(session, participant, NULL, NULL);
+					if(!first && participant->codec == JANUS_AUDIOCODEC_OPUS && lost_packets_gap <= JITTER_BUFFER_MAX_GAP_SIZE) {
+						lost_packets_gap += 1;
+
+						janus_mutex_unlock(&participant->qmutex);
+						locked = FALSE;
+
+						if(!g_atomic_int_compare_and_exchange(&participant->decoding, 0, 1)) {
+							/* This means we're cleaning up, so don't try to decode */
+							janus_audiobridge_buffer_packet_destroy(bpkt);
+							break;
+						}
+
+						int32_t output_samples;
+						opus_decoder_ctl(participant->decoder, OPUS_GET_LAST_PACKET_DURATION(&output_samples));
+
+						pkt = g_malloc(sizeof(janus_audiobridge_rtp_relay_packet));
+						pkt->data = g_malloc0(BUFFER_SAMPLES * sizeof(opus_int16));
+						pkt->ssrc = 0;
+						pkt->timestamp = participant->last_timestamp + OPUS_SAMPLES;
+						pkt->seq_number = participant->last_seq + 1;
+						/* This is a redundant packet, so we can't parse any extension info */
+						pkt->silence = FALSE;
+						janus_audiobridge_participant_istalking(session, participant, NULL, NULL);
+						pkt->length = opus_decode(participant->decoder, NULL, 0, (opus_int16 *)pkt->data, output_samples, 0);
+#ifdef HAVE_RNNOISE
```

Comment:
do we need to run this on plc packets?

Reply:
I think so, because PLC packets are generated from some sort of interpolation from the encoded audio, I believe, that we may have cleaned out of noise in previous iterations after decoding it. Not denoising the PLC audio could result in discontinuities in the audio.